<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcWindow</class>
 <widget class="QMainWindow" name="CalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CalcWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLineEdit" name="InputDisplay">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	color: #615F5F;
	margin-bottom: 30px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>128</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Type a math problem</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::VisualMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="arithm">
       <attribute name="title">
        <string>+ -</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="3">
         <widget class="QPushButton" name="Button3">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="Button9">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="Button8">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="ButtonDot">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="ButtonOpenBracket">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="Button1">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="Button4">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="ButtonCloseBracket">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="ButtonDiv">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="ButtonX">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="Button6">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="ButtonEqual">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>=</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="ButtonMinus">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="ButtonMult">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="Button2">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="Button5">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="Button0">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="ButtonPow">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>^</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="ButtonPlus">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="Button7">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="funcs">
       <attribute name="title">
        <string>sin cos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QPushButton" name="ButtonAcos">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: white;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>acos</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="ButtonSin">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: white;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>sin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="ButtonCos">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: white;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>cos</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="ButtonAtan">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: white;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>atan</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="ButtonAsin">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: white;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>asin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="ButtonTan">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: white;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>tan</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="ButtonLog">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: white;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>log</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="ButtonLn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: white;
	color: black;
}</string>
          </property>
          <property name="text">
           <string>ln</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="ButtonSqrt">
          <property name="text">
           <string>sqrt</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="ButtonMod">
          <property name="text">
           <string>mod</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="ButtonGraph">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Graph</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="ButtonMemoryRemove">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="ButtonMemoryAdd">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="ButtonMemoryGet">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="ButtonBackspace">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #fff;
	border: 1px solid gray;
	border-radius: 4px;
	background-color: white;
	color: black;
}

QPushButton::pressed {
	background-color: #D5D2D2;
	border: 1px solid gray;
	border-radius: 4px;
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Backspace</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
